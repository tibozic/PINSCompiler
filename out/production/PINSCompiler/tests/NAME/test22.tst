# program ki presteje stevilo sodih stevil
fun main ( tab : arr[20] integer ) : integer = (
	{for i = 0, 20, 1:
		{if tab[i] % 2 == 0 then
			{stsodih = stsodih + 1}
		}
	}
) {where var stsodih : integer}
