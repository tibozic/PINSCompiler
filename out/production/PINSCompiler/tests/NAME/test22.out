[3:10-3:11]: ERROR: No definition found for `i`

